src/status_im/chat/views/actions.cljs

```diff
@@ -114,12 +114,12 @@
         subtitle])]]])
 
 (defn actions-list-view []
```

Comment:
`actions-list-view` doesn't conform re-frame's form 2
maybe it is better to replace it with `defview`